test/search.cc

```diff
@@ -254,6 +254,17 @@ void test_edge_search() {
   search({test}, false, answer,
          {PE{{t, l, 3}, ratio, answer, 0, S::RIGHT}, PE{{t, l, 8}, 1.f - ratio, answer, 0, S::LEFT}});
 
+  // try getting side of street by setting the display ll instead
+  x = {answer};
+  x.display_latlng_ = test;
+  search(x, false, answer,
+         {PE{{t, l, 3}, ratio, answer, 0, S::RIGHT}, PE{{t, l, 8}, 1.f - ratio, answer, 0, S::LEFT}});
+
+  // try display ll on the other side
+  x.display_latlng_ = {answer.first - ortho.x(), answer.second - ortho.y()};
+  search(x, false, answer,
+         {PE{{t, l, 3}, ratio, answer, 0, S::LEFT}, PE{{t, l, 8}, 1.f - ratio, answer, 0, S::RIGHT}});
+
```

Comment:
what we do here is we take a test for side of street that was relying on the routing ll to determine side of street and we put the routing ll on the road and then set the display ll to what was the routing ll. in this case the display ll is offset perpendicular to the road. we test a display ll on both sides of the road.